tests/unittests/test_error_handling.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+import requests
+from tap_tiktok_ads.client import TikTokAdsClientError, TikTokClient
+from unittest import mock
+
+# mocked response class
+class Mockresponse:
+    def __init__(self, status_code, json, headers=None):
+        self.status_code = status_code
+        self.text = json
+        self.headers = headers
+
+    def json(self):
+        return self.text
+
+# function to get mocked response
+def get_response(status_code, json={}, headers=None):
+    return Mockresponse(status_code, json, headers)
+
+# @mock.patch("time.sleep")
+@mock.patch("requests.Session.request")
+class TestInternalErrorBackoff(unittest.TestCase):
```

Comment:
Change the class name as these test cases are for error messages and not for backoff.

Reply:
Changed the class name